opentelemetry-stackdriver/src/lib.rs

```diff
@@ -731,6 +732,24 @@ impl From<EvictedHashMap> for Attributes {
     }
 }
 
+fn transform_links(links: &EvictedQueue<opentelemetry::trace::Link>) -> Option<Links> {
+    if links.is_empty() {
+        return None;
+    }
+
+    Some(Links {
+        dropped_links_count: links.dropped_count() as i32,
+        link: links
+            .iter()
+            .map(|link| Link {
+                trace_id: hex::encode(link.span_context.trace_id().to_bytes()),
+                span_id: hex::encode(link.span_context.span_id().to_bytes()),
```

Comment:
Can we have `INVALID` IDs here, and if so, should we treat them differently?

Reply:
So, the `SpanBuilder` does check the validity of links, but if you add links by hand later, you could add links with invalid spans contexts. But I don't think this is something which should be cought here. Maybe the span builder should add an API and make the `links` field private. Enforcing the validity check in the new added API.
Note that the check above is only needed because currently SpanData uses the invalid span id to mark the span as a root span.